x-pack/spec/filters/geoip/download_manager_spec.rb

```diff
@@ -86,26 +94,38 @@ module LogStash module Filters module Geoip
         expect(download_manager).to receive(:md5).and_return(md5_hash)
 
         path = download_manager.send(:download_database, db_info)
-        expect(path).to match /GeoLite2-City_\d+\.mmdb\.gz/
+        expect(path).to match /GeoLite2-City_\d+\.tgz/
         expect(::File.exist?(path)).to be_truthy
-        ::File.delete(path) if ::File.exist?(path)
+
+        delete_file(path)
       end
     end
 
     context "unzip" do
-      before(:each) do
+      let(:copyright_path) { get_file_path('COPYRIGHT.txt') }
+      let(:license_path) { get_file_path('LICENSE.txt') }
+      let(:readme_path) { get_file_path('README.txt') }
+
+      before do
         file_path = ::File.expand_path("./fixtures/sample", ::File.dirname(__FILE__))
-        ::File.delete(file_path) if ::File.exist?(file_path)
+        delete_file(file_path, copyright_path, license_path, readme_path)
       end
 
-      it "gz file" do
-        path = ::File.expand_path("./fixtures/sample.gz", ::File.dirname(__FILE__))
-        unzip_path = download_manager.send(:unzip, path)
-        expect(::File.exist?(unzip_path)).to be_truthy
+      it "should extract database and license related files" do
+        path = ::File.expand_path("./fixtures/sample.tgz", ::File.dirname(__FILE__))
+        unzip_db_path = download_manager.send(:unzip, path)
+
+        expect(unzip_db_path).to match /\.mmdb/
+        expect(::File.exist?(unzip_db_path)).to be_truthy
+        expect(::File.exist?(copyright_path)).to be_truthy
+        expect(::File.exist?(license_path)).to be_truthy
+        expect(::File.exist?(readme_path)).to be_falsey
+
+        delete_file(unzip_db_path, copyright_path, license_path)
       end
     end
 
-    context "assert database" do
+    xcontext "assert database" do
```

Comment:
Because we move the`GeoIPFilter.java` from `org.logstash.filters` to `org.logstash.filters.geoip` in ECS support PR.
Once the plugin release a new version, this test case will reopen again